inspektor/lint.py

```diff
@@ -64,6 +64,7 @@ def get_opts(self):
             if self.args.enable:
                 pylint_args.append('--enable=%s' % self.enabled_errors)
             pylint_args.append('--reports=no')
+            pylint_args.append('--errors-only')
```

Comment:
Right, I forgot to consider that errors only actually removed all messages that are not stricly errors. I'm dumb like that :)

I just wanted to remove the annoying grade message at the end of every check. I'll keep trying.